### DISCUSSION TITLE
[下载视频 - WASM 混流输出] 能否支持视频批量下载？

### DISCUSSION THREAD

**Discussion by Temp314159** [the1812/Bilibili-Evolved#4734] [功能建议 / Ideas]
以前，我要下载B站视频，只能在手机APP下载后导出音视频文件到电脑再ffmpeg合并。直到——我发现Bilibili-Evolved提供了下载视频组件，还能一键混流输出！
泪，流了下来！
 
然鹅，今天，当我勾选了多P视频，试图批量下载多个视频时，一个令人悲伤的提示弹了出来：
“[WASM 混流输出] 仅支持单个视频下载”
我的泪，再次流了下来！
 
感觉这样的使用场景还是比较常见的，比如一些分了几十P的教学视频。
其它输出方式似乎都不提供混流？总之就是无法一键下载多个合并后的视频……
（我不会说我以前下载多个视频时都是1P1P点下来的……然而遇到N多P的视频臣妾实在做不到TAT）

**Comment by WakelessSloth56** [comment]
#4726 已经合并了，下一版本就有了。

（@the1812 能否帮忙改一下插件的描述，加上支持多集，写PR时我忘记了。）
